ratis-metrics/src/main/java/org/apache/ratis/metrics/RatisMetrics.java

```diff
@@ -31,9 +31,16 @@
   protected RatisMetricRegistry registry;
 
   protected static RatisMetricRegistry create(MetricRegistryInfo info) {
+    return create(info, true);
+  }
+
+  protected static RatisMetricRegistry create(MetricRegistryInfo info,
+      boolean shouldDebugLog) {
     Optional<RatisMetricRegistry> metricRegistry = MetricRegistries.global().get(info);
     return metricRegistry.orElseGet(() -> {
-      LOG.info("Creating Metrics Registry : {}", info.getName());
+      if (shouldDebugLog) {
```

Comment:
Why not `LOG.isDebugEnabled()`?

Reply:
This mertics is used by different components. Currently we were seeing it create a lot of noise in client logs. Using this `shouldDebugLog` helps with not printing the message for client logs (even in Debug mode) but printing for other components such as LeaderElectionMetrics. 

For the other components, we can keep it Info or Debug level. I am ok either way.

Reply:
I understand the motivation but for the sake of the simplicity I would vote to always use LOG.debug level if `LOG.isDebugEnabled` (and drop the specific parameter). I think the information here is not so important to have an explicit flag for it. If somebody were interested, debug log level always can be adjusted.

(But I am fine with either approach)

Reply:
bq. I think the information here is not so important to have an explicit flag for it. If somebody were interested, debug log level always can be adjusted.

Agreed. I will remove the extra flag and keep the LOG at debug level.